pkg/systemd/overview-cards/realmd-operation.js

```diff
@@ -602,7 +603,8 @@ export function setup() {
                 self.dispatchEvent("changed");
             }
             $(realmd).off();
-            realmd.close();
+            if (realmd)
+                realmd.close();
```

Comment:
At this point in the code the `realmd` variable should never be `null`. I figure what happens is that calling setup_realms_proxy() a second time does not clean up the initial `"close"` event handler, so that will still run, set `realmd = null`, and crash the second handler.

So I think a better fix would be to call `$(realmd).off()` at the start of the function if `realmd` is already defined, to clean up the old event handler.